src/lib/components/ControlModal.svelte

```diff
@@ -68,6 +68,9 @@ function change(idx: number, ev: Event) {
 	<ModalHeader {toggle}>Controls in Scope</ModalHeader>
 	<ModalBody>
 		<div class="container">
+			<!-- Hide add/remove of controls if catalog is EUCS  -->
+			<!-- TODO(all): Use catalog.inScope or something else for checking the scope -->
+			{#if target.catalogId != "EUCS"}
```

Comment:
you can use catalog.allInScope to check this

Reply:
Yes, I thought of that. But at this point, I don't have the catalog, do I? And I didn't know now exactly how to get it, so I did it that way.

Reply:
The data of all catalogs is available in `data.catalogs`. You need to loop through it to find the one with the correct ID. In the future, we might also use a svelte store for this, as we do for the metrics. This is very inconsistent in the moment and needs to be changed (but with a lower priority).